.prettierrc.json

```diff
@@ -1,7 +1,7 @@
 {
+  "trailingComma": "es5",
```

Comment:
is this config recommended by team? changes on this config adding unnecessary changes in other files.

Reply:
reverted this back to `"trailingComma": "all"` @saugatmaharjan